apps/web/src/components/editor/index.tsx

```diff
@@ -382,7 +382,9 @@ function EditorView({
         }}
         options={{
           readonly: session?.type === "readonly" || session?.type === "deleted",
-          onRequestFocus: () => toggleProperties(false),
+          onRequestFocus: () => {
+            toggleProperties(false, false);
+          },
```

Comment:
Instead of all these changes, let's just remove this "close on focus" logic altogether.